src/View/Helper/FormHelper.php

```diff
@@ -253,6 +253,11 @@ public function control($fieldName, array $options = [])
                         $options['templates']['checkboxContainer'] = '{{content}}';
                     }
                 }
+
+                $options['templates']['nestingLabel'] = '{{hidden}}{{input}}<label{{attrs}}>{{text}}</label>';
```

Comment:
Since BS4 doesn't need nested inputs for checkbox and radio instead of setting this template it would be better to modify relevant code to simply not use nesting labels.

Reply:
if `nestedInput` is false, the input doesn't appear at all, same issue that @ndm2 had

Reply:
Well than let's fix the cause rather than the symptom. :slightly_smiling_face:

Reply:
isn't that a core problem? one which might someday get fixed in cake4? relying on nesting labels for now and updating the template to put the input outside is at least a workaround until then

Reply:
BUI's form helper overrides core's so we can make the necessary fix here. Once I understand the source of the problem I can fix core too.